packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -115,6 +120,10 @@ class HelpGroup {
                     return e;
                 }),
             },
+            {
```

Comment:
I think we should have negated flags with `Options` only, as to the user they are flags only.

Reply:
To be clear, spreading these with `optionList` of `Options` header is what I am suggesting here.

Reply:
![Screenshot at 2020-07-07 11-16-07](https://user-images.githubusercontent.com/46647141/86721341-84b4ba80-c043-11ea-983f-92e4870c7a9e.png)

Reply:
like this ?

Reply:
Yes